examples/react_webpack/tsconfig.json

```diff
@@ -4,6 +4,11 @@
         "lib": [
             "ES2015",
             "DOM"
+        ],
+        "plugins": [
+            {
+                "transform": "transformer-module"
```

Comment:
does this do something? I'm expecting something to declare a dependency on this package

Reply:
I'll try to replicate what's being done here: https://github.com/cevek/ttypescript/tree/master/packages/ttypescript-examples

And I'll add a test to make sure the outputted .js is what we expect after the transformation

Reply:
Got blocked by https://github.com/cevek/ttypescript/issues/110. Removing the transformation example for now